src/lib/arcx.ts

```diff
@@ -0,0 +1,21 @@
+import { ArcxAnalyticsSdk } from '@arcxmoney/analytics'
+
+const ARCX_API_KEY = ''
+
+let client: ArcxAnalyticsSdk
+
+export async function getArcxClient() {
```

Comment:
why not turn this into a hook?

Reply:
because im lazy :D . will do, good callout 🙏